default/AI/FreeOrionAI.py

```diff
@@ -174,9 +174,9 @@ def generateOrders():  # pylint: disable=invalid-name
     print "**********   String for chart. Do not modify.   ***************************"
     print ("Generating Orders")
     print ("EmpireID: {empire.empireID}"
-           " Name: {empire.name}_{empire.empireID}_pid:{p_id}_{p_name}RIdx_{res_idx}_{aggression}"
+           " Name: {empire.name}_{empire.empireID}_pid:{p_id}_{p_name}_{aggression}"
```

Comment:
Didn't look into details (I didn't even realize that is related to "charting" stuff) :sweat: 
Reading the charting codes seems to tell me that charting should remain working (beyond a name change, because new research AI no longer have a notion of "research index")